core/src/plugins/filed/python/vmware/BareosFdPluginVMware.py

```diff
@@ -889,95 +1005,101 @@ def prepare_vm_backup(self):
         - get disk devices
         """
         if "uuid" in self.options:
-            vmname = self.options["uuid"]
```

Comment:
Is uuid handling still required?

Reply:
As mentioned above, the uuid option still works as in the old version, but restore to other dc, folder, datastore is not possible and trying it will be rejected by the plugin now. Added that in docs.